File: Sprint/Sprint/Models/TaskDTO.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace Sprint.Models
{
    public class TaskDTO
    {
        public Guid TaskId { get; set; }
        public string TextTask { get; set; }

        public Guid FunctionalityId { get; set; }

    }
}

File: FrontendTemplateProject-main/src/app/models/UserStoryModel/taskDto.ts
import { Guid } from 'guid-typescript';

export class TaskDto {
    
    taskId!: Guid;
    textTask!: string;
    functionalityId!: Guid;
    
    constructor(
        taskId: Guid,
        textTask: string,
        functionalityId: Guid
    ) {
        this.taskId = taskId;
        this.textTask = textTask;
        this.functionalityId = functionalityId;
    }
    
}